package.json

```diff
@@ -148,8 +148,20 @@
     "stylelint-processor-styled-components": "^1.10.0",
     "typescript": "^4.9.5",
     "typescript-coverage-report": "^0.7.0",
-    "vite": "^5.4.8",
+    "vite": "^6.1.0",
     "vite-plugin-svgr": "^4.2.0",
-    "vitest": "^3.0.2"
+    "vitest": "^3.0.6"
+  },
+  "pnpm": {
+    "overrides": {
```

Comment:
I had to add the overrides, because there was no way to force the upgrade of these two packages by upgrading the ones that we install directly that require them to the minimal, required version